client/src/components/SiteSearch/SiteSearchListing/SiteSearchListing.js

```diff
@@ -29,17 +36,53 @@ CMSListingItem.propTypes = {
   }).isRequired
 };
 
-export const FileListingItem = ({ name, size, lastModified, path }) => (
-  <div styleName="sitesearch-cms-item">
-    <h5>{name}</h5>
-    <div>{path}</div>
-  </div>
-);
-FileListingItem.propTypes = {
-  name: PropTypes.string.isRequired,
-  size: PropTypes.number.isRequired,
-  lastModified: PropTypes.string.isRequired,
-  path: PropTypes.string.isRequired
+export const SiteSearchFileListing = ({ listing, filter }) => {
+  const fileNavCellCallback = useCallback(
+    ({ row }) => {
+      return (
+        <FileNavCell
+          system={row.original.system}
+          path={row.original.path}
+          name={row.original.name}
+          format={row.original.format}
+          api="tapis"
+          scheme={filter}
+          href={row.original._links.self.href}
+          isPublic={filter === 'public'}
+        />
+      );
+    },
+    [filter]
+  );
+
+  const tableColumns = [
+    {
+      id: 'icon',
+      accessor: 'format',
+      Cell: FileIconCell
+    },
+    {
+      accessor: 'name',
+      Cell: fileNavCellCallback,
+      className: 'td-width-100'
+    },
+    { accessor: 'length', Cell: FileLengthCell, className: 'td-no-overflow' }
+  ];
+
+  return (
+    <>
+      {filter}
+      <InfiniteScrollTable
```

Comment:
Something discussed in implementation of onboarding admin - if you have a paginator (because number of total results are available) then infinite scroll table may or may not be the best option here. Good for style consistency, though.

Reply:
AFAIK an infinite sroll table without a scroll callback is just a table, right?

Reply:
Yes, though @tacc-wbomar has made extensive style customizations to InfiniteScrollTable.

Reply:
I am confused.

1. Are `<InfiniteScrollTable>` and `<Paginator>` incompatible?
2. Can they become compatible (I see "custom" pagination exists here, with `<InfiniteScrollTable>`).

If we need to choose one or the other to complete this ticket, I would vote to:
- **both** keep `<InfiniteScrollTable>` (because there are already too many table inconsistencies)
- **and** make _either_ a ticket to resolve the incompatibility _or_ an explanation I can add to Confluence¹

> ¹ [UI - Behavior / UX - Tables](https://confluence.tacc.utexas.edu/x/D4BCCQ) or [WIP: UI - Components - Table](https://confluence.tacc.utexas.edu/x/gwD9Bw)